src/Image/Image.php

```diff
@@ -144,9 +144,9 @@ public function html(array $attr = []): string
 	/**
 	 * Returns the PHP imagesize array
 	 */
-	public function imagesize(): array
+	public function imagesize(): array|false
 	{
-		return getimagesize($this->root);
+		return @getimagesize($this->root);
```

Comment:
`Question`: @afbora Do we need the `@` to suppress errors? Or would it work without as well. The docs say `On failure, false is returned.`